File: Models/Article.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Backend_Foto.Models
{


    public class Article
    {
        public int ID { get; set; }
        public string Title { get; set; }
        public string Author { get; set; }
        public string Content { get; set; }
        public string Date { get; set; }
        public string Img { get; set; }
        public int? View { get; set; }

        //public Article(int id, string title, string author, string content, string date, string img, int view)
        //{
        //    ID = id;
        //    Title = title;
        //    Author = author;
        //    Content = content;
        //    Date = date;
        //    Img = img;
        //    View = view;

        //}

        //public Article(int iD, string title, string author, string content, string date, string img, int? view)
        //{
        //    ID = iD;
        //    Title = title;
        //    Author = author;
        //    Content = content;
        //    Date = date;
        //    Img = img;
        //    View = view;
        //}
    }
}

File: ClientApp/src/models/Article.ts
export class Article {
  ID: number;
  Title: string;
  Content: string;
  Author:string;
  Date: string;
  Img: string;
  View: number;
  AuthorID: number;
  constructor( tittle: string,author:string, content: string, date: string, img:string, authorID:number) {
    this.Title = tittle;
    this.Author=author;
    this.Content = content;
    this.Date = date;
    this.Img=img;
    this.AuthorID=authorID;
  }
}
